src/parser/parsers.h

```diff
@@ -1314,11 +1314,17 @@ template<typename Ctx> Result<> makeRefCast(Ctx& ctx, Index pos) {
 }
 
 template<typename Ctx> Result<> makeBrOnNull(Ctx& ctx, Index pos, bool onFail) {
```

Comment:
I'm missing where the Contexts are created and how they are passed into the "make" series of functions.

Reply:
The contexts are created and passed to the top-level parse function here: https://github.com/WebAssembly/binaryen/blob/main/src/parser/wat-parser.cpp#L96-L169